timeless/restaurants/table_shapes/views.py

```diff
@@ -32,24 +34,15 @@ def list():
         table_shapes=query.all())
 
 
-@bp.route("/create", methods=("GET", "POST"))
-def create():
-    """ Create new table shape"""
-    form = forms.TableShapeForm(request.form)
-
-    if request.method == "POST" and form.validate():
-        """
-        @todo #162:30min This form currenly cannot save pictures. Need have a
-         look how WTF form processes files. Implement generic solution to use
-         it everywhere when it's needed.
-        """
-        """
-        Uncomment after correction
-        form.save()
-        """
-        return redirect(url_for("table_shape.list"))
-    return render_template(
-        "restaurants/table_shapes/create_edit.html", form=form)
+class Create(views.CreateView):
+    """
+    @todo #162:30min This form currenly cannot save pictures. Need have a
```

Comment:
@vryazanov why can't it process images, what exactly is the exception? Furthermore, I don't think the Form has to process images, but only paths to them? 

It depends how you wanna do it. I would personally only save the path to the image in the DB (and store the Image in some other way, not through this form). For instance, when you upload files manually to Github, it sends them to AWS S3 behind the scenes and Github itself only stores paths.

Reply:
@amihaiemil I absolutely agree with you. There is should be only path in the db, but the logic of receiving multipart/formdata should be still finished.
We just need to use the code from https://flask-wtf.readthedocs.io/en/stable/form.html#module-flask_wtf.file and check that it works.
btw it's not a new puzzle, it already was here